components/InputsOne.js

```diff
@@ -0,0 +1,64 @@
+import { useRef, useEffect, useState } from "react";
+import styled from "styled-components";
+
+export default function InputsOne({
+  canvasOneParameter,
+  handleColorChange,
+  handleRotationChange,
+  handleOutlineChange,
+  handleVariation,
+}) {
+  return (
+    <StyledInputWrapper>
+      <label htmlFor="colorSelector">Color </label>
+      <StyledInputRange
+        type="range"
+        min="0"
+        max="360"
+        value={canvasOneParameter[0].color}
+        onChange={(event) => handleColorChange(event.target.value)}
+        id="colorSelector"
+      />
+      <label htmlFor="rotation">Rotation </label>
+      <StyledInputRange
+        type="range"
+        min="0"
+        max="90"
+        value={canvasOneParameter[1].rotation}
+        onChange={(event) => handleRotationChange(event.target.value)}
+        id="rotation"
+      />
+      <label htmlFor="outline">Amount of outlines </label>
+      <StyledInputRange
+        type="range"
+        min="0"
+        max="8"
+        value={canvasOneParameter[2].outline}
+        onChange={(event) => handleOutlineChange(event.target.value)}
+        id="outline"
+      />
+      <StyledVariatonButton onClick={() => handleVariation()}>
```

Comment:
You should add `type="button"` as a button is by default a submit button.

Reply:
Added! :)